flowmachine/flowmachine/core/server/query_schemas/unique_subscriber_counts.py

```diff
@@ -0,0 +1,45 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marshmallow import Schema, fields, post_load
+from marshmallow.validate import OneOf, Length
+
+from flowmachine.features import UniqueSubscriberCounts
+from .base_exposed_query import BaseExposedQuery
+from .custom_fields import AggregationUnit
+
+__all__ = ["UniqueSubscriberCountsSchema", "UniqueSubscriberCountsExposed"]
+
+
+class UniqueSubscriberCountsSchema(Schema):
+
+    start_date = fields.Date(required=True)
+    end_date = fields.Date(required=True)
+    aggregation_unit = AggregationUnit()
+
+    @post_load
+    def make_query_object(self, params):
+        return UniqueSubscriberCountsExposed(**params)
+
+
+class UniqueSubscriberCountsExposed(BaseExposedQuery):
+    def __init__(self, *, start_date, end_date, aggregation_unit):
+        # Note: all input parameters need to be defined as attributes on `self`
+        # so that marshmallow can serialise the object correctly.
+        self.start_date = start_date
+        self.end_date = end_date
+        self.aggregation_unit = aggregation_unit
+
+    @property
+    def _flowmachine_query_obj(self):
+        """
+        Return the underlying flowmachine daily_location object.
```

Comment:
```suggestion
        Return the underlying flowmachine UniqueSubscriberCounts object.
```

Reply:
I've implemented the suggested alternative, but that _flowmachine_query_ob j() method was already present and the only change I made was to fix the object name in theader. So that won't change the test coverage. Presumably something must be added to actually call _flowmachine_query_obj() ?! (I assumed some test would run it automatically, as its name looks pretty generic.)

Reply:
@OwhHute This particular suggestion about changing `daily_location` to `UniqueSubscriberCounts` in the docstring is independent of @greenape's comment about code coverage and the need to add an additional test. (It may look related because Github displays it right underneath, but it has nothing to do with it).